### ISSUE TITLE
Help in the Expression Builder

### ISSUE THREAD

**Issue by Arturoe1** [0x90d/videoduplicatefinder#409]
**Environment**
 - OS: [e.g. Windows / Linux / Any] Any

**Describe the solution you'd like**
The custom expression builder does not provide enough information. 
For example if you want to use :  arg.ItemInfo.FrameSize, nobody knows what is expected exactly aside for an string. 
Can we user greater than or less than?  What kind of string is expecting? 

This happens to many expressions as the help provided is just 3 examples. 

Is it possible to get a proper help in the app or here so we know what is expected in each expression?

**Comment by Topspap** [comment]
Yes more explanation of tutorial on how to use the "Expression Builder" could be really helpful as it should be a powerful way to automate the selection and speed up the process.
Thank you...
